modules/custom-status/custom-status.php

```diff
@@ -1501,6 +1568,7 @@ private function get_preview_link( $post ) {
 		} else if ( 'post' == $post->post_type ) {
 			$args = array(
 					'p'          => $post->ID,
+					'preview'	 => 'true'
```

Comment:
Unsure if this is needed for the other post_type's, but the unit tests based off of core require this. Maybe add a unit test for page or a custom post_type.